agent-tooling/src/main/java/io/opentelemetry/auto/tooling/DefaultExporterConfig.java

```diff
@@ -16,7 +16,7 @@
 
 package io.opentelemetry.auto.tooling;
 
-import io.opentelemetry.auto.config.Config;
+import io.opentelemetry.instrumentation.library.api.config.Config;
```

Comment:
Since we don't have `library` in our artifact names for library, guess we wouldn't have it in this package name?

Reply:
makes sense to me, done